MpsKeySearchDevice/Makefile

```
@@ -0,0 +1,12 @@
+NAME=MpsKeySearchDevice
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)